ale_linters/markdown/redpen.vim

```diff
@@ -0,0 +1,32 @@
+" Author: rhysd https://rhysd.github.io
+" Description: Redpen, a proofreading tool (http://redpen.cc)
+
+function! ale_linters#markdown#redpen#HandleErrors(buffer, lines) abort
+    " Only one file was passed to redpen. So response array has only one
+    " element.
+    let l:res = json_decode(join(a:lines))[0]
+    let l:errors = []
+    for l:err in l:res.errors
+        if has_key(l:err, 'startPosition')
+            let l:lnum = l:err.startPosition.lineNum
+            let l:col = l:err.startPosition.offset
```

Comment:
While you're at it, could you add a condition below for reading the `endPosition` too? You can set the `end_lnum` and `end_col` keys. Then we'll be able to draw better error highlights.

Reply:
sure.